integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -241,3 +245,55 @@ def get_drafted_pages(
             "drafted_pages": drafted_pages,
             "single_drafted_page": single_drafted_page,
         }
+
+    @json_response
+    # pylint: disable=unused-argument, disable=no-self-argument
+    def get_translation_coverage_context(
+        request: HttpRequest, region_slug: str
+    ) -> JsonResponse:
+        r"""
+        Extend context by info on translation coverage of pages
+
+        :return: Dictionary containing the context for translation coverage of pages in a region
+        """
+        number_of_outdated_pages = 0
+
+        if not request.region.default_language:
+            return {}
+
+        languages = LanguageTreeNode.objects.filter(region=request.region).exclude(
+            language=request.region.default_language
+        )
+
+        pages_in_region = Page.objects.filter(
+            region=request.region, explicitly_archived=False
+        )
+        for page in pages_in_region:
```

Comment:
Somehow the for loop doesn't catch implicitly archived pages right. When I test it locally with the data of 'Stadt Augsburg' it doesn't count the one implicitly archived page in the data

Reply:
The approach of @charludo seems to solve this problem, since the same approach seems to solve a similar problem in refactoring the page tree #3130